serialization.go

```diff
@@ -82,44 +82,42 @@ func Send(w http.ResponseWriter, r *http.Request, ans any) error {
 	case "application/xml":
 		return SendXML(w, r, ans)
 	case "text/html":
-		return SendHTML(r.Context(), w, ans)
+		return SendHTML(w, r, ans)
 	case "text/plain":
-		return SendText(w, ans)
+		return SendText(w, r, ans)
 	case "application/json":
-		SendJSON(w, ans)
+		return SendJSON(w, r, ans)
 	case "application/x-yaml", "text/yaml; charset=utf-8", "application/yaml": // https://www.rfc-editor.org/rfc/rfc9512.html
-		SendYAML(w, ans)
+		return SendYAML(w, r, ans)
 	default:
 		return errors.New("unsupported Accept header")
 	}
```

Comment:
I assume this is just preference?

Reply:
I always find better to

it's a simple convention to put the default at the end, but unless I'm wrong, (but [I don't think so](https://goplay.space/#Ctzlw3o958b))  "default" is always evaluated at the end when they are other "cases", but these cases are evaluated in the exact order they are. See my example https://goplay.space/#Ctzlw3o958b

Why am I talking about that ?

Because some people might add a case after the default, making the code hard to read.
Having a return at the end instead of a default help to keep things clear.

But yes, it's a personal preference, only based on code reviews and people ability to surprise me with strange code.